docs/documentation/changelog.md

```diff
@@ -78,11 +75,11 @@ denote the various action types:![add](../media/img/add.jpg)=add,
 <img alt="update" src="../media/img/update.jpg" />=update
 ***
 
-{% include changelog/42.1.1-release.md %}
-
-{% include changelog/42.1.0-release.md %}
-
-{% include changelog/42.0.0-release.md %}
+{% for post in site.categories.new_release %}
+<a name="version_{{ post.version }}"></a>
+## Version {{ post.version }} ({{ post.date | date: "%Y-%d-%m"}})
+{{ post }}
+{% endfor %}
```

Comment:
Hmmm.... yes it does look quite bad

https://jdbc.postgresql.org/changelog.html